tests/wicked/startandstop_sut.pm

```diff
@@ -102,13 +102,42 @@ sub run {
     my $static_ip = $self->get_ip(type => 'host', no_mask => 1);
     my $dhcp_ip = $self->get_current_ip($iface);
     if (defined($dhcp_ip) && $static_ip ne $dhcp_ip) {
-        $results{5} = $self->get_test_result('host');
+        $results{5} = $self->get_test_result("host");
```

Comment:
why ? as far as I know single quotes is preferable and you should use double quotes only when you have some variable int the string

Reply:
just to keep it equal to the other places where we call this function, but not important...

Reply:
will be ok for me if you will change 'other places' to use single quotes instead :)

Reply:
can be in same PR but please do it as separate commit

Reply:
sure